omod/src/main/webapp/resources/scripts/emr.js

```diff
@@ -87,6 +87,22 @@ var emr = (function($) {
             }
         },
 
+        serverValidationErrorMessage: function(response) {
+        	var errorMessage = "";
+        	if (response.data != null && response.data.error != null) {
+        		errorMessage = response.data.error.message;
+        	}
+        	return errorMessage;
+        },
+
+        serverGeneralErrorMessage: function(response) {
+        	var errorMessage = "";
+        	if (response.data != null && response.data.error != null) {
```

Comment:
As for these, i think this is correct, sometimes the message can have sensitive data, so displaying a generic message i guess is fine.
There is no point i having this if clause since you have use the data